website/src/components/Edit/EditPage.tsx

```diff
@@ -101,31 +66,33 @@ const InnerEditPage: FC<EditPageProps> = ({
     );
 
     const submitEditedDataForAccessionVersion = () => {
+        const metadataFile = editableMetadata.getMetadataTsv(dataToEdit.submissionId, dataToEdit.accession);
+        if (metadataFile === undefined) {
+            toast.error('Please enter metadata.', { position: 'top-center', autoClose: false });
```

Comment:
Not necessarily blocking, but this seems a bit weird. Theoretically one could have have no metadata fields I guess - ideally it feels like the logic for this should be in the backend (which yes, I guess would already require a dummy metadata file with `submissionIds`, but potentially only that?)

Reply:
I could imagine that this might only appear if they enter no submission id?

Reply:
this is also resolved I think?